src/en/ss14-by-example/ui-and-you.md

```diff
@@ -0,0 +1,474 @@
+-   `StyleIdentifier`: The identifier of the element. This is a unique identifier
+    that can be used to target a specific element. This should be used sparingly,
+    when there is only one instance of the element that needs to be styled in a
+    highly specific manner. There may only be one of these specified.
+-   `PseudoClasses`: These are special classes that can be used to target elements
+    in a specific state. For example, this is used to style buttons differently
+    when hovered or pressed or whatever.
+
+Selectors that specify more of these parts are more "specific", and will take
+priority over less specific elements.
+
+**The properties**: Any elements matching the selector will then have their properties
+modified by the style rule. These are the same properties you would define in
+the XAML.
+
+To assist with constructing these style rules, there are helper methods defined in
+`Content.Client/Stylesheets/StylesheetHelpers`.
+
+I think it's best to show an examples of style rules instead of describing all
+their intricacies since the code is pretty self-explanatory.
+
+```cs
+// you need this using statement to use the helper methods
+using static Content.Client.Stylesheets.Redux.StylesheetHelpers;
+
+var rules =
+[
+    // select any element...
+    E()
+        // ...with the class "negative"
+        .Class(StyleClass.Negative)
+        // ...and set its font color to the text color from the negative palette
+        .FontColor(sheet.NegativePalette.Text),
+
+    // select any `Label`...
+    E<Label>()
+        // ...with the class "LabelHeading"
+        .Class(StyleClass.LabelHeading)
+        // ...and set its font to a bold 16pt font
+        .Font(sheet.BaseFont.GetFont(16, FontKind.Bold))
+        // ...and its font color to the text color from the highlight palette
+        .FontColor(sheet.HighlightPalette.Text)
+
+    // select any `ContainerButton`...
+     E<ContainerButton>()
+        // ...with the class "button"
+        .Class(ContainerButton.StyleClassButton)
+        // ...and the class "ButtonSmall"
+        .Class(StyleClass.ButtonSmall)
+        // ...that is the parent of a `Label`,
+        .ParentOf(E<Label>())
+        // ...and set that `Label`'s font to an 8pt font
+        .Font(sheet.BaseFont.GetFont(8))
+];
+```
+
+### Death to Hardcoding!
+
+You may have noticed before that there wasn't much hardcoding in the rule definitions.
+This is because most style rules are used in multiple stylesheets, and they have
+to be generic between them. There are a couple different utilities used to reduce
+hardcoding.
+
+#### `ColorPalette`
+
+There is actually a pretty robust (haha) color palette system to hopefully make
+hardcoding colors unnecessary. I created a
+[codepen](https://codepen.io/aspiringLich/pen/VwOXdjd?editors=1000)
+to help visualize the colors. The stylesheets then use these palettes to set the following
+common palettes for sheetlets to reference:
```

Comment:
This last sentence is weird. The stylesheets use palettes to set palettes? What does it means for sheetlets to reference something? I think it should be more direct about what actual effect palettes have.